comps/embeddings/langchain-mosec/requirements.txt

```diff
@@ -0,0 +1,12 @@
+docarray[full]
+fastapi
+#huggingface_hub
```

Comment:
shall you remove those lines which are commented out.

Reply:
Yep, sure. I will modify it.